index.php

```diff
@@ -16,7 +16,8 @@
 }
 
 //301 to add trailing slash
-if (substr($uri, -1) !== '/') {
+$segments = explode('/', $uri);
+if (substr($uri, -1) !== '/' && strpos(end($segments), ".") === false) {
     $parts = explode('?', $_SERVER['REQUEST_URI'], 2);
     $protocol = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on') ? "https" : "http";
     $redirectUri = $protocol . '://' . $_SERVER[HTTP_HOST] . $parts[0] . '/' . (isset($parts[1]) ? '?' . $parts[1] : '');
```

Comment:
Bring in globals.php and replace $_SERVER[HTTP_HOST] with HTTP_HOST. since everything is a proxy  and $_SERVER[HTTP_HOST] will output "localhost"

Reply:
Done.